app/javascript/packs/leafletMap.js

```diff
@@ -1,12 +1,19 @@
 let map;
 let routeLayer;
+let layerControl;
+// Defines the floor that is currently displayed
+let currentFloor = 0;
+// object that contains for each existing floor two layer groups, one for rooms and one for labels
+const floors = {};
```

Comment:
What about making this a [Map](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Map)?

Reply:
I have thought about it but could not see a clear reason to switch in the end. Why would you prefer a map here?

Reply:
the only advantage i see is having number as keys

Reply:
I will not change it to the Map in this pull request however it might be the better option in the future